### DISCUSSION TITLE
Help With connecting Arduino Uno

### DISCUSSION THREAD

**Discussion by KingcornholioField** [tttapa/Control-Surface#1103] [Q&A]
I have followed all the steps to get my uno found a midi device, but in the arduino IDE I am unable to select the board and it doesn't show up. Anybody have any help or ideas?

**Comment by tttapa** [comment]
Which steps did you follow? If you replaced the ATmega16U2 firmware, you'll have to restore the original Arduino firmware before you can program it over Serial again.

https://github.com/arduino/ArduinoCore-avr/tree/master/firmwares/atmegaxxu2

**Reply by KingcornholioField** [reply]
Okay, I got it thanks!
